### PR TITLE
 Chapter 11 Account activation

### PR DESCRIPTION
**Pull request by ptrang97**
image

### CODE REVIEW

**Comment by ptrang97** [comment]
ready

File: app/assets/javascripts/account_activations.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```
Comment (phamvanbk11) [inline app/assets/javascripts/account_activations.coffee]: Xóa file

File: app/assets/stylesheets/account_activations.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the AccountActivations controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (phamvanbk11) [inline app/assets/stylesheets/account_activations.scss]: Xóa file

File: app/assets/javascripts/account_activations.coffee (hunk 1 above)
Comment (ptrang97) [reply app/assets/javascripts/account_activations.coffee]: đã fix ạ

File: app/assets/stylesheets/account_activations.scss (hunk 2 above)
Comment (ptrang97) [reply app/assets/stylesheets/account_activations.scss]: đã fix ạ

**Comment by ptrang97** [comment]
ready

**Comment by phamvanbk11** [comment]
commented

**Comment by ptrang97** [comment]
ready

File: app/controllers/sessions_controller.rb [hunk 3]
```diff
@@ -1,15 +1,10 @@
...
 class SessionsController < ApplicationController
   def new; end
 
   def create
     user = User.find_by email: params[:session][:email].downcase
-    if user&.authenticate params[:session][:password]
-      log_in user
-      remember_check params[:session][:remember_me], user
-      redirect_to user
+    if user && user.authenticate(params[:session][:password])
```
Comment (VuThiTranVan) [inline app/controllers/sessions_controller.rb]: ` user && user.authenticate` -> ` user&.user.authenticate` nhé gái

File: test/controllers/account_activations_controller_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
```
Comment (VuThiTranVan) [inline test/controllers/account_activations_controller_test.rb]: xóa file

File: app/controllers/sessions_controller.rb (hunk 3 above)
Comment (ptrang97) [reply app/controllers/sessions_controller.rb]: đã fix ạ

File: test/controllers/account_activations_controller_test.rb (hunk 4 above)
Comment (ptrang97) [reply test/controllers/account_activations_controller_test.rb]: đã fix ạ

**Comment by ptrang97** [comment]
ready

**Comment by phamvanbk11** [comment]
merged

**Pull request merged**